AzureTable/Microsoft.DataTransfer.AzureTable/Source/AzureTableLocationMode.cs

```diff
@@ -0,0 +1,30 @@
+﻿
+namespace Microsoft.DataTransfer.AzureTable.Source
+{
+    /// <summary>
+    /// Azure Table storage internal fields enumeration.
+    /// </summary>
+        
+    public enum AzureTableLocationMode
+    {
+        /// <summary>
+        /// Primary Only
```

Comment:
I like to have extensive descriptions, like "`Connect to primary end-point only.`", but at least use dot "`.`" at the end to be consistent with the rest of the codebase.